### DISCUSSION TITLE
Loading Enums on the front end

### DISCUSSION THREAD

**Discussion by ccsv** [reset-button/ariadne_django#60] [Q&A]
@maxmorlocke 

Do you have to write a resolver to load the results for a enum query on the frontend?

I am querying values like this and I am having a difficult time getting the results.

```
query Category {
    __type(name: "category") {
      enumValues {
        name
      }
    }
  }
```

**Comment by maxmorlocke** [comment]
Can you share a bit more of your code so I can understand on what's tripping up?  Is name the enum?  I'm not really sure what's going on here.

**Reply by ccsv** [reply]
Sorry to bother you with this but I am trying to figure out the enums. Ok So I have a django model like this

```
class Category(models.IntegerChoices):
  '''Categories'''
  Open = 1
  Pending = 2
  Completed = 3
```

and I have the types as:

```
type_defs = gql("""
enum Category {
    Open
    Pending
    Completed
  }"""
  )
  
 class Category(enum.IntEnum):
    Open = 1
    Pending= 2
    Completed = 3

category_enum = EnumType("Category", Category)

schema = make_executable_schema(type_defs, query, mutation, category_enum)
```

I am trying to access the enums on the frontend (like for dropdown options for a form) Do I need to write a resolver for this? I am using the query from above in vue-apollo and I am not sure if I need to write a resolver or I am doing something wrong on the front end.
